### ISSUE TITLE
UI Navigation - Studies: Grouping of Studies of same Patient not always works

### ISSUE THREAD

**Issue by gunterze** [dcm4che/dcm4chee-arc-light#3036]
Grouping of Studies of same Patient not always works:
![Screenshot from 2021-02-08 11-24-13](https://user-images.githubusercontent.com/400095/107207687-e6d20580-6a00-11eb-932b-d5f7eb4c35a4.png)

Send attached objects to archive to reproduce the issue:
[SCSGERM.zip](https://github.com/dcm4che/dcm4chee-arc-light/files/5943093/SCSGERM.zip)

**Comment by vrindanayak** [comment]
@shral : Remember from a discussion we had long back - UI orders the studies of same patient based on `Study Date` received in responses. Unsure, if this is still the case.

**Comment by shral** [comment]
@gunterze Can not reproduce the bug with the attached zip (there is only one study in there)!

**Comment by gunterze** [comment]
Are you sure, you sent both images to the archive? There Study Instance UID differs:
```console
$ dcmdump /tmp/1.2.840.113674.950809133358028.100.dcm | grep 000D
890: (0020,000D) UI #28 [1.2.840.113674.1115.261.200] StudyInstanceUID
```
```console
$ dcmdump /tmp/SCSGERM | grep 000D
626: (0020,000D) UI #40 [1.3.6.1.4.1.5962.1.2.0.1175775772.5723.0] StudyInstan
```

**Comment by gunterze** [comment]
@shral Verify if you have already an MR Study 1.3.6.1.4.1.5962.1.2.0.1175775772.5723.0 associated to a different Patient in your archive. If so, reject that study and delete the patient permanently, before sending 1.2.840.113674.950809133358028.100.dcm

**Comment by shral** [comment]
There is a difference in the patient attributes in bouth studies:

![image](https://user-images.githubusercontent.com/7990614/107794556-e1a1ed00-6d57-11eb-886d-ba789d77a100.png)

**Comment by gunterze** [comment]
Solved by #3049
